lib/pacemaker/pcmk_sched_promotable.c

```diff
@@ -135,9 +135,11 @@ apply_promoted_locations(pe_resource_t *child,
 
             pe_rsc_trace(child,
                          "Applying location %s to %s promotion priority on %s: "
-                         "%d + %d = %d",
+                         "%s + %s = %s",
```

Comment:
If you want we can either get rid of this equation notation here, or use it in other places where we put the new score in a temp variable (so we have the old and the new) -- e.g., `pcmk__add_promotion_scores()`). I have no preference right now